definitions/npm/enzyme_v2.3.x/flow_v0.28.x-/enzyme_v2.3.x.js

```diff
@@ -1,82 +1,104 @@
-declare module 'enzyme' {
-  declare type PredicateFunction<T: Wrapper> = (wrapper: T, index: number) => boolean;
+declare module "enzyme" {
+  declare type PredicateFunction<T: Wrapper> = (
+    wrapper: T,
+    index: number
+  ) => boolean;
   declare type NodeOrNodes = React$Element<any> | Array<React$Element<any>>;
-  declare type EnzymeSelector = string | ReactClass<any> | Object;
+  declare type EnzymeSelector = string | Class<React$Component<*>> | Object;
```

Comment:
I think it might be better to use [`React$ComponentType<*>`](https://flow.org/en/docs/react/types/#toc-react-componenttype), so you can use either class components or stateless components.